webapp/i18n/es.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
For my understanding: This file does **not** need to be in the bundle, correct?

Reply:
Correct. It could be extracted using file-loader like in the webapp, but for simplicity I just imported it directly here. For most plugins this is probably fine as they won't have a lot of translations.

Reply:
I see. :+1: Importing it directly also seams the easier way here.